subprojects/packagefiles/kafel/meson.build

```
@@ -0,0 +1,140 @@
+project(
+  'kafel',
+  'c',
+  version: '20231004',
+  license: 'Apache-2.0',
+  default_options: ['c_std=gnu11'],
+)
+
+if host_machine.system() != 'linux'
+  error('kafel is not supported on this platform.')
+endif
+
+if host_machine.system() == 'linux' and host_machine.cpu_family() == 'ppc64le'
+  error('kafel does not support ppc64le')
+endif
```

The endianness is [not included](https://mesonbuild.com/Reference-tables.html#cpu-families) in the CPU family, and the magic words for this case are `unsupported architecture`.

```suggestion
if host_machine.cpu_family() == 'ppc64'
  error('unsupported architecture: ppc64')
endif
```